lib/diagnostics.ts

```diff
@@ -23,21 +23,21 @@ declare global {
 window.Twilio = window.Twilio || {};
 window.Twilio.Diagnostics = {
   ...window.Twilio.Diagnostics,
+  testAudioInputDevice,
+  testAudioOutputDevice,
   testBitrate,
-  testInputDevice,
-  testOutputDevice,
 };
 
 /**
  * Expose the tests if installed as a npm module for both TS and JS
  */
 export {
+  AudioInputTest as AudioInputTest,
```

Comment:
Just curious, why do we need to do this?

Reply:
We don't need it. I think vscode automatically added it. I updated the PR and removed it.